src/apps/clients/vice.clj

```diff
@@ -0,0 +1,22 @@
+(ns apps.clients.vice
+  (:require [cemerick.url :as curl]
+            [cheshire.core :as cheshire]
+            [clj-http.client :as http]
+            [apps.util.config :as cfg]
+            [clojure.tools.logging :as log]))
+
+(defn- app-exposer-url
+  [& components]
+  (str (apply curl/url (cfg/app-exposer-base-url) components)))
+
+(defn stop-job
+  [uuid]
+  (http/post (app-exposer-url "vice" uuid "save-and-exit")
+             {:throw-exceptions false
+              :as               :json}))
+
+(defn submit-job
+  [job]
+  (http/post (app-exposer-url "vice" "launch")
+             {:content-type :json
+              :body         (cheshire/encode job)}))
```

Comment:
You can replace `:body (cheshire/encode job)` with `:form-params job` if you want. clj-http will automatically do the JSON encoding as long as the content type is `:json`. This isn't a requirement or anything, just a convenience that I use a lot.